src/App.js

```diff
@@ -1,26 +1,296 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+    import React, { Component } from 'react';
+
+
+    import './App.css';
+    import Table from './table/Table'
+    import Navigation from './table/Navigation'
+
+    class App extends Component {
+
+
+      state = {
+
+
+        showTable: false,
+
+
+        showApp: "Void",
+        showTests: "Void",
+        showEnv: "Void",
+        testCount: 14,
+
+
+
+
+        testResults: [ { release: "001", name: "", date: "", app:
```

Comment:
Let's pull this data out into a `constants` file. The idea being that this muddies up the file so its hard to see the actual code. We do this for Cypress tests, where we import the constants/test data